app/javascript/datasources/index.js

```diff
@@ -48,3 +48,25 @@ export const postJobsAuthenticate = async (password) => {
 
     return await result.json();
 };
+
+export const testDonationAmounts = [
```

Comment:
@jemmaissroff I think we originally said 10, 20, 50, 100... but an even number of prices looks better than an odd one so I switched it to 10, 25, 50, 75, 100... Let me know if that looks okay to you.